contrib/gitian-descriptors/gitian-windows.yml

```diff
@@ -23,23 +23,18 @@ remotes:
 - "url": "https://github.com/cosmos/gaia.git"
   "dir": "gaia"
 files:
-- "golang-debian-1.12.5-1.tar.gz"
+- "go1.14.linux-amd64.tar.gz"
 script: |
   set -e -o pipefail
 
-  GO_SRC_RELEASE=golang-debian-1.12.5-1
+  GO_SRC_RELEASE=go1.14.linux-amd64
```

Comment:
ditto

Reply:
We've rolled out an entirely new reproducible buildsystem, so I wouldn't bother too much with this cause it will go away soon.